library.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Is this additional file really needed? I don't want to spam my repository.

Reply:
As far as I'm aware - it is :(

Reply:
Yes, I confirm. Without the proposed config, PIO tries to build files in all subfolders (including examples) and ends with errors. As a temporary workaround, I had to make a local copy of the headers.